src/main/java/com/rits/cloning/Cloner.java

```diff
@@ -586,6 +603,17 @@ public IFreezableCloner(Class<?> clz) {
 			instantiator = instantiationStrategy.getInstantiatorOf(clz);
 		}
 
+		private boolean isFieldNullInsteadBecauseOfAnnotation(Field f) {
+			for (Annotation annotation : f.getAnnotations()) {
```

Comment:
to make this more performant, can you please add some code to check if nullInsteadFieldAnnotations.isEmpty() then don't do anything

Reply:
Good point. I added the required check.